Index: app/battery_manager/routes.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from app.battery_manager import bp\n\n\n@bp.route('/')\ndef main_page():\n    return 'Main page of batteries manager'\n\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/battery_manager/routes.py b/app/battery_manager/routes.py
--- a/app/battery_manager/routes.py	(revision 5513ee1e705f73f95307a2f6dc1ca982cf3a8cff)
+++ b/app/battery_manager/routes.py	(date 1685301359571)
@@ -1,4 +1,7 @@
 from app.battery_manager import bp
+from flask import render_template
+from flask import request
+from app.services.add_battery import process_form
 
 
 @bp.route('/')
@@ -6,3 +9,13 @@
     return 'Main page of batteries manager'
 
 
+@bp.route('/add', methods=['POST', 'GET'])
+def add_new_battery():
+    if request.method == 'POST':
+        process_form(request)
+    return render_template('battery_manager/main_page.html',
+                           template_folder='templates'
+                           )
+
+
+
